docs/UserGuide.adoc

```diff
@@ -161,51 +202,120 @@ Examples:
 
 * `show 2`
 
-=== Import/Export
-==== Importing eateries: `import`
+===== Deleting an eatery: `delete`
+
+Delete an eatery in the application. +
+Format: `delete 1`
+
+==== Visualisation
+===== Showing overall statistics: `stats`
 
 *Coming in v1.4*
 
-Imports eateries from a text file. +
-Format: `import \f [filename]`
+Presents a visual overview of the user’s eateries and reviews. +
+Format: `stats`
+
+
+=== Todo Mode
+
+In Todo Mode, manipulating of eateries which user had yet to visit will be available. +
```

Comment:
In Todo mode, users will be able to manipulate eateries which they have yet to visit.